src/utils/link-channel.js

```diff
@@ -62,6 +62,7 @@ export default class LinkChannel {
               if (this.store.state.activity.hasChangedName) {
                 data.profile = { ...this.store.state.profile };
               }
+              data.credentials = { ...this.store.state.credentials };
```

Comment:
we probably need to bump the code to 5 or even 6 letters with this change now

Reply:
(yes non-scientific, but /shrug)

Reply:
I dunno, already seems like a low enough probability for now given our traffic.

Reply:
oh, the mechanism we're missing actually that would help here is a second confirmation step on the link workflow back on the original device. that doesn't have to gate this functionality but that is relatively high priority to ensure people can't snipe credentials.  i'll add it to github